challenge.js

```diff
@@ -57,5 +57,5 @@
  */
 
 // ANSWER
-
+http://www.w3schools.com/jsref/met_node_appendchild.asp
```

Comment:
No w3schools! Use MDN! This is not a personal preference, w3schools is a lower-quality resource for JS documentation.

The method is correct, though. 1